base-requirements.txt

```diff
@@ -22,32 +22,32 @@ chardet==4.0.0
 celery[redis]==5.3.6
 django-celery-beat==2.5.0
 # TODO: We may drop 'django-imagekit' completely.
-django-imagekit==4.0.2
+django-imagekit==5.0  # 5.0 is first version that supports Django 4.2
 django-haystack==3.2.1
 elasticsearch>=7,<8
 # TODO: 0.14.0 only supports Django 1.8 and 1.11.
-django-tastypie==0.14.3
+django-tastypie==0.14.6  # 0.14.6 is first version that supports Django 4.2
 
 pytz==2021.1
 python-dateutil==2.8.2
 
 requests[security]>=2.26.0
 
-django-honeypot==1.0.1
-django-markupfield==2.0.0
-django-markupfield-helpers==0.1.1
+django-honeypot==1.0.4  # 1.0.4 is first version that supports Django 4.2
+django-markupfield==2.0.1
+# django-markupfield-helpers==0.1.1  # this hasnt been moved out of alpha and is last published 2018, we do import it
```

Comment:
i cant find where we use this, but it seems we dont. anyway, it doesnt support either 3.2 or 4.2 iirc

Reply:
This was used in `sponsors/pdf.py` and `sponsors/tests/test_pdf.py` which no longer exist.  I created a PR to remove the dependency:
* https://github.com/python/pythondotorg/pull/2474

You can either wait for my PR to be approved/merged and then merge `main` into this branch, or just remove it directly from this branch:
```suggestion
```